pkgs/development/haskell-modules/configuration-nix.nix

```diff
@@ -720,6 +720,17 @@ self: super: builtins.intersectAttrs super {
     '';
   }) super.sbv;
 
+  rest-rewrite = overrideCabal (drv: {
+    testToolDepends = (drv.testToolDepends or [ ]) ++ [ pkgs.graphviz pkgs.z3 ];
```

Comment:
Would you be able to send a PR to `cabal2nix` adding this?

For example, here:

https://github.com/NixOS/cabal2nix/blob/078350047d358bb450d634d775493aba89b21212/cabal2nix/src/Distribution/Nixpkgs/Haskell/FromCabal/PostProcess.hs#L73C64-L73C64

In this function you should see some examples of adding new test tools.  You should add a new line for `rest-rewrite`.

Reply:
Ah, I wondered where that information came from. Thanks, will do.